statistics/ddl.go

```diff
@@ -136,14 +131,26 @@ func (h *Handle) insertColStats2KV(tableID int64, colInfo *model.ColumnInfo) err
 		if err != nil {
 			return errors.Trace(err)
 		}
-		value, err = value.ConvertTo(h.ctx.GetSessionVars().StmtCtx, types.NewFieldType(mysql.TypeBlob))
-		if err != nil {
-			return errors.Trace(err)
-		}
-		// There must be only one bucket for this new column and the value is the default value.
-		_, err = exec.Execute(fmt.Sprintf("insert into mysql.stats_buckets (table_id, is_index, hist_id, bucket_id, repeats, count, lower_bound, upper_bound) values (%d, 0, %d, 0, %d, %d, X'%X', X'%X')", tableID, colInfo.ID, count, count, value.GetBytes(), value.GetBytes()))
-		if err != nil {
-			return errors.Trace(err)
+		if value.IsNull() {
```

Comment:
Add some comment:
"If adding column with default value null, all the existing rows have null value on the newly added column."